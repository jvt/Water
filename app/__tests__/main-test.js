jest.disableAutomock();

const React = require('React');
const Renderer = require('react-test-renderer');
const Button = require('Button');

describe('ProfileButton', () => {
    it('renders correctly', () => {
        const instance  = Renderer.create(
            <Button 
            style={{}}
            title="Profile">
            Profile
            </Button>
        );

        expect(instance.toJSON()).toMatchSnapshot();
    });
});