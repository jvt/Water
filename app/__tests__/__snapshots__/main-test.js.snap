exports[`ProfileButton renders correctly 1`] = `
<View
  accessibilityComponentType="button"
  accessibilityLabel={undefined}
  accessibilityTraits={
    Array [
      "button",
    ]
  }
  accessible={true}
  hitSlop={undefined}
  isTVSelectable={true}
  onLayout={undefined}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "opacity": 1,
    }
  }
  testID={undefined}
  tvParallaxProperties={undefined}>
  <View
    style={
      Array [
        Object {},
      ]
    }>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "color": "#0C42FD",
            "fontSize": 18,
            "padding": 8,
            "textAlign": "center",
          },
        ]
      }>
      Profile
    </Text>
  </View>
</View>
`;
